integration/js/ContentSecurityPolicyTest.js

```diff
@@ -0,0 +1,38 @@
+const {OpenAppStep, JoinMeetingStep, AuthenticateUserStep, WaitForMeetingToBeCreated, WaitForRemoteParticipantsToJoinMeeting} = require('./steps');
+const {UserAuthenticationCheck, SecurityPolicyViolationCheck, UserJoinedMeetingCheck, RosterCheck} = require('./checks');
+const {AppPage} = require('./pages/AppPage');
+const {TestUtils} = require('./node_modules/kite-common');
+const SdkBaseTest = require('./utils/SdkBaseTest');
+const { v4: uuidv4 } = require('uuid');
+
+/*
+* 1. Starts a meeting
+* 2. Adds a participants to the meeting
+* 3. Checks if there are security policy violation event
+* */
```

Comment:
Why add a new test for this? Each of these tests is extraordinarily expensive when considered in aggregate. Why not add the security policy violation check to the Voice Focus test, which is where we are most likely to hit CSP issues?